x-pack/plugins/observability/public/pages/overview/containers/overview_page/overview_page.tsx

```diff
@@ -65,7 +68,6 @@ export function OverviewPage() {
 
   const { data: newsFeed } = useFetcher(() => getNewsFeed({ http }), [http]);
   const { hasAnyData, isAllRequestsComplete } = useHasData();
-  const refetch = useRef<() => void>();
```

Comment:
@kdelemme Since you reviewed my PR, I wanted a second opinion about this `useRef`, I didn't see any reason to keep it, please let me know if you see something that I am missing.

Reply:
I feel like this was used for retuning something from the useCallback, but return nothing from the useCallback is perfectly fine. So in my opinion, this ref is unused and provides no value. 👍🏻